src/ai/backend/manager/api/session.py

```diff
@@ -1438,18 +1438,15 @@ async def get_abusing_report(request: web.Request, params: Mapping[str, Any]) ->
 )
 async def sync_agent_registry(request: web.Request, params: Any) -> web.StreamResponse:
     root_ctx: RootContext = request.app["_root.context"]
-    session_name: str = request.match_info["session_name"]
     requester_access_key, owner_access_key = await get_access_key_scopes(request)
 
-    log.info(
-        "SYNC_AGENT_REGISTRY (ak:{}/{}, s:{})", requester_access_key, owner_access_key, session_name
-    )
+    log.info("SYNC_AGENT_REGISTRY (ak:{}/{})", requester_access_key, owner_access_key)
     try:
-        await root_ctx.registry.sync_agent_kernel_registry(params["agent"])
+        await root_ctx.registry.sync_agent_kernel_registry(AgentId(params["agent"]))
     except BackendError:
         log.exception("SYNC_AGENT_REGISTRY: exception")
         raise
-    return web.json_response({}, status=200)
+    return web.json_response({}, status=201)
```

Comment:
In this case, we should keep "200 OK" instead of "201 Created", because we have not added anything.